docs/source/howto/data.rst

```diff
@@ -260,6 +260,359 @@ Finding data
+
+From the python interface:
+
+.. code-block:: ipython
+
+    In [1]: group = load_group(label='test_group')
+
+    In [2]: group.remove_nodes([load_node(1)])
+
+Alternatively, you might want to remove *all* nodes from the group.
+In the command line you just need to add ``-c/--clear`` option to ``verdi group remove-nodes ..``
+
+.. code-block:: console
+
+    $ verdi group remove-nodes -c -G test_group
+    Do you really want to remove ALL the nodes from Group<test_group>? [y/N]:
+
+In the python interface you can use ``.clean()`` method to achieve the same goal:
+
+.. code-block:: ipython
+
+    In [1]: group = load_group(label='test_group')
+
+    In [2]: group.clear()
+
+
+Rename Group
+^^^^^^^^^^^^
+From the command line interface:
+
+.. code-block:: console
+
+      $ verdi group relabel test_group old_group
+      Success: Label changed to old_group
+
+From the python interface:
+
+.. code-block:: ipython
+
+    In [1]: group = load_group(label='old_group')
+
+    In [2]: group.label = 'another_group'
+
+
+Delete Group
+^^^^^^^^^^^^
+From the command line interface:
+
+.. code-block:: ipython
+
+      $ verdi group delete another_group
+      Are you sure to delete Group<another_group>? [y/N]: y
+      Success: Group<another_group> deleted.
+
+.. note::
+    Any deletion operation related to groups won't affect the nodes themselves.
+    For example if you delete a group, the nodes that belonged to the group will remain in the database.
+    The same happens if you delete nodes from the group -- they will remain in the database but won't belong to the group anymore.
+
+Copy one group into another
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+This operation will copy the nodes of the source group into the destination group.
+If the destination group does not yet exist, it will be created automatically.
+
+From the command line interface:
+
+.. code-block:: console
+
+    verdi group copy source_group dest_group
+    Success: Nodes copied from group<source_group> to group<dest_group>
+
+From the python interface:
+
+.. code-block:: ipython
+
+    In [1]: src_group = Group.objects.get(label='source_group')
+
+    In [2]: dest_group = Group(label='destination_group').store()
+
+    In [3]: dest_group.add_nodes(list(src_group.nodes))
+
+
+Organizing data with groups
+---------------------------
+
+In this section, we will provide some practical examples of how one can use Groups to structure and organize the nodes in the database.
+
+Case I: grouping structures with a similar property
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Suppose you have a database of structures for which you have performed a set of simulations.
+Your goal is to continue the study only for the structures for which the bandgap is higher than ``1.0 eV``
+To achieve that goal you would first need to make a query that finds such structures and puts them into a group.
+Let's go through the whole process step by step.
```

Comment:
Again, this should not be a tutorial. Users are not expected to follow along.

Reply:
We can then move this section somewhere else. Is there a better place for it to be?

Reply:
Do you mean all of the showcase examples?

Reply:
yes, everything below ` Examples for using groups `

Reply:
I'm not sure there is since we agreed that there would be only one tutorial for now. Is my other suggestion – to convert this into a how-to – not acceptable?